kaggle_environments/static/player.html

```diff
@@ -561,20 +576,22 @@
                 if (
                   interactive ||
                   isInteractive() ||
-                  (key !== 32 && key !== 37 && key !== 39)
+                  (key !== 32 && key !== 37 && key !== 39 && !(key >= 48 && key <= 57))
```

Comment:
nit: I think you could just remove this clause and put an `else` at the end of the `if` chain right below?  But then I feel like the `event.preventDefault()` might never be able to be called which seems weird and I'm probably misreading...

Also I know I did not follow through when you made a similar recommendation for my Halite renderer PR, but consider adding constants for these key values :)

Reply:
I don't think so since the outer boolean condition is an `||` but maybe I'm not understanding which clause you mean